utils/pom.xml

```diff
@@ -59,7 +59,12 @@
         </dependency>
         <dependency>
             <groupId>za.co.absa</groupId>
-            <artifactId>spark-commons_${scala.compat.version}</artifactId>
+            <artifactId>spark-commons-spark${spark.compat.version}_${scala.compat.version}</artifactId>
+            <version>${absa.spark.commons.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>za.co.absa</groupId>
+            <artifactId>spark-commons-test_${scala.compat.version}</artifactId>
             <version>${absa.spark.commons.version}</version>
         </dependency>
```

Comment:
At first glance, I would expect this dependency to be scoped to `test` only, but I understand that compile-scoped `testUtils` currently prevent you from doing so.

Perhaps some other time in the future, when we drop mvn support as it would get easier.

Reply:
Indeed, on both accounts.